Gaijinizer patch/Map061.txt

```diff
@@ -138,48 +151,53 @@ No, I'm sure E-003 will get lost ....#MTLed
 ただでさえ劣勢な人間たちに、俺たちはみすみすヤバい脅威を
 新たに送り込むことになっちまうんだぜ…？
 > CONTEXT: Map061/events/5/pages/0/93/Dialogue/other2
-But what happens then? Even to the inferior
-people, we helplessly. What do you want to do?#MTLed
+But what happens then?
+
+At best, we've just knowingly sent in a dangerous
+new threat against humanity...#Ron
 > END STRING
 
 > BEGIN STRING
 極めつけは、E-003の再生能力が異常すぎて
 
 肝心の『安全装置』すら効かねぇって話じゃねぇか…
 > CONTEXT: Map061/events/5/pages/0/100/Dialogue/other2
-At the very end, the playback ability of E-003 is
-too abnormal. Even if the important "safety
-device" is effective, it is not talk about it ...#MTLed
+And the cherry on top is E-003's abnormally
+powerful regenerative ability. All the "compliance
+devices" in the world ain't gonna be able
+to keep it down...#Ron
```

Comment:
Cleanup of line limits:
>_**And the cherry on top is E-003's abnormally
powerful regenerative ability. All the 
"compliance devices" in the world ain't
 gonna be able to keep it down...**_